climbing_stairs/climbing_stairs.py

```diff
@@ -3,7 +3,16 @@
 import sys
 
 def climbing_stairs(n, cache=None):
-  pass 
+  if n <= 1:
+    return 1
+  elif n == 2: 
+    return 2
+  elif n == 3:
+    return 3
+  else:
+    return climbing_stairs(n-3) + climbing_stairs(n-2) + climbing_stairs(n-1)
+
```

Comment:
You'll want an `elif` to check if the cache exists and if the cache entry at `n` is greater than zero, which returns the cache entry at `n`. Also a way to create the cache if it's not there, and return the cache entry which is equal to what you have here on line 13.